tools/cd_scripts/install_test.sh

```diff
@@ -62,39 +72,41 @@ else
     echo "Failure detected in latest gcsfuse version installation." &>> ~/logs.txt
 fi
 
-# Uninstall gcsfuse latest version and install old version
-if grep -q ubuntu details.txt || grep -q debian details.txt;
-then
-    sudo apt remove -y gcsfuse
-    sudo apt install -y gcsfuse=0.42.5 -t gcsfuse-$(lsb_release -cs) |& tee -a ~/logs.txt
-else
-    sudo yum -y remove gcsfuse
-    sudo yum -y install gcsfuse-0.42.5-1 |& tee -a ~/logs.txt
-fi
+if [[ $architecture == amd64 ]]; then
```

Comment:
why do we have this if ? (there is no else to it also)

Reply:
This if block will execute only for amd64 machines as we don't have any previous release for arm64.

Reply:
added comment for the same.

Reply:
This part will also not execute https://github.com/GoogleCloudPlatform/gcsfuse/blob/a5f5782386123da077a7d34d9615b02cf22eda66/tools/cd_scripts/install_test.sh#L98? And we should execute this part ?

Reply:
Once we will release one version, we will add that. At this point I don't think so it is require.

Reply:
my concern is that customers can still run upgrade command. So, we should ensure that the command works (even though the version already exists)
Also, please make note of things on internal bug what code changes we should do from next release.

Reply:
Yes will add.